src/MultiplayerMod/Multiplayer/Chores/ChoresPatcher.cs

```diff
@@ -0,0 +1,124 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using HarmonyLib;
+using JetBrains.Annotations;
+using MultiplayerMod.Core.Collections;
+using MultiplayerMod.Core.Events;
+using MultiplayerMod.Core.Extensions;
+using MultiplayerMod.Core.Logging;
+using MultiplayerMod.Core.Scheduling;
+using MultiplayerMod.Game.Chores;
+using MultiplayerMod.ModRuntime;
+using MultiplayerMod.ModRuntime.Context;
+using MultiplayerMod.ModRuntime.StaticCompatibility;
+using MultiplayerMod.Multiplayer.Chores.Commands;
+using MultiplayerMod.Multiplayer.Chores.Events;
+using MultiplayerMod.Multiplayer.CoreOperations;
+using MultiplayerMod.Multiplayer.Objects.Extensions;
+using MultiplayerMod.Multiplayer.StateMachines;
+using MultiplayerMod.Network;
+
+namespace MultiplayerMod.Multiplayer.Chores;
+
+[Core.Dependency.Dependency, UsedImplicitly]
+public class ChoresPatcher {
+
+    private static EventDispatcher events = null!;
+    private static IMultiplayerServer server = null!;
+    private static List<Type> choreTypes = [];
+
+    private readonly Harmony harmony;
+    private readonly Core.Logging.Logger log = LoggerFactory.GetLogger<ChoresPatcher>();
+    private readonly ConditionalWeakTable<ChoreDriver, BoxedValue<bool>> driverSynchronizationState = new();
+
+    public ChoresPatcher(
+        EventDispatcher events,
+        Harmony harmony,
+        StateMachinesPatcher patcher,
+        IMultiplayerServer server
```

Comment:
Shall we avoid sending events here and rise event instead? E.g. chore creation raises event while choreSet send command directly.